client/commands.cc

```diff
@@ -261,13 +262,21 @@ command_create_post_xsnapshot(DBus::Connection& conn, const string& config_name,
 
 void
 command_delete_xsnapshots(DBus::Connection& conn, const string& config_name,
-			  list<unsigned int> nums)
+			  list<unsigned int> nums, bool verbose)
 {
     DBus::MessageMethodCall call(SERVICE, OBJECT, INTERFACE, "DeleteSnapshots");
 
     DBus::Hoho hoho(call);
     hoho << config_name << nums;
 
+    if (verbose) {
+	cout << "Deleting snapshots: ";
+	for (list<unsigned int>::iterator it = nums.begin(); it != nums.end(); ++it) {
+	    cout << *it << ",";
```

Comment:
This will result in a trailing comma. You can check with it != nums.begin() and not output a comma before the first number, see https://github.com/openSUSE/snapper/blob/master/snapper/SnapperTmpl.h#L70. I also prefer a whitespace after the comma.